frontend/packages/core/src/AppProvider/index.tsx

```diff
@@ -91,29 +91,35 @@ const ClutchApp: React.FC<ClutchAppProps> = ({
         <div id="App">
           <ApplicationContext.Provider value={{ workflows: discoverableWorkflows }}>
             <Routes>
-              <Route path="/*" element={<AppLayout isLoading={isLoading} />}>
-                <Route key="landing" path="/" element={<Landing />} />
+              <Route path="/" element={<AppLayout isLoading={isLoading} />}>
+                <Route path="/" element={<Landing />} />
```

Comment:
This seems a bit strange but you can't have `/*` and then `/`. But it seems to extend correctly as you go down the tree.

Reply:
I'd be surprised if we didn't need the wildcard. The [React Router docs](https://reactrouter.com/docs/en/v6/getting-started/overview#descendant-routes) definitely say it's required
```
If you do this, make sure to put a * at the end of the parent route's path. Otherwise, the parent route won't match the URL when it is longer than the parent route's path, and your descendant <Routes> won't ever show up.
```

Reply:
Yeah this is the error from the console:

"Absolute route path "/" nested under path "/*" is not valid. An absolute child route path must start with the combined path of all its parent routes."

So basically we have:

/ - parent
/ - landing
/<workflow> - workflows
/* - notFound
* - notFound

Reply:
Adjusting to remove the path from the landing route and keep it nested under "/*"